corehq/apps/integration/static/integration/js/payments/payments_verify.js

```diff
@@ -0,0 +1,34 @@
+import "commcarehq";
+import "hqwebapp/js/htmx_and_alpine";
+import $ from "jquery";
+import { multiCheckboxSelectionHandler } from "integration/js/checkbox_selection_handler";
+
+
+function updateVerifyButton(selectedIds) {
+    const $verifyBtn = $('#verify-selected-btn');
+    let verifyBtnVals = JSON.parse($verifyBtn.attr('hx-vals'));
+
+    if (selectedIds.length > 0) {
+        $verifyBtn.prop('disabled', false);
+        verifyBtnVals['selected_ids'] = selectedIds;
+    } else {
+        $verifyBtn.prop('disabled', true);
+        verifyBtnVals['selected_ids'] = [];
+    }
```

Comment:
Since `selectedIds` has been updated at this point, we could make this a bit more DRY and replace the entire conditional with the following:

```suggestion
    $verifyBtn.prop('disabled', !Boolean(selectedCount))
    verifyBtnVals['selected_ids'] = selectedIds;  // This will either contain a list of IDS, or be an empty list
```

Reply:
Addressed in https://github.com/dimagi/commcare-hq/commit/d7c7a06cbcf74ff28fc806322fafae58362203f9